### DISCUSSION TITLE
What is the data returned from experimental `useWriteContracts`

### DISCUSSION THREAD

**Discussion by DobromirKirovLime** [wevm/wagmi#4248] [Q&A] [answered]
I am really confused with the returned data from this hook. In the docs is stated 'Most recent block number seen.' which i can understand, but how to format or read something from it.  

example response: `0x507097c0e9670e7c0b763003c14ae78f29a3234214345a789591fb44d82409a40000000000000000000000000000000000000000000000000000000000002105`
How can i approach this, can i extract tx data from it?

**Comment by johanneskares** [comment, marked as answer]
https://wagmi.sh/react/api/hooks/useCallsStatus
